### PR TITLE
PART 1 - Exception when exiting from App with no editors

### PR DESCRIPTION
**Pull request by OSKozlov**
**Problem:** Exception when exiting from App with no editors. Parent get disposed due to event queue clearing in BrowserPanel.handleEvent. while(Display.getDefault().readAndDispatch()).

**Solution:** Add check for widget is disposed.

Part 2 - https://github.com/Halliburton-Landmark/eclipse.platform.ui/pull/31/

### CODE REVIEW

**Comment by OSKozlov** [comment]
Reviewer: @IlyaMikhailovHB

File: bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/IE.java [hunk 1]
```diff
@@ -954,6 +954,7 @@ else if (IEVersion >= 8) {
 					if (arg13.getType() == OLE.VT_BSTR) {
 						String text = arg13.getString();
 						StatusTextEvent newEvent5 = new StatusTextEvent(browser);
+						if (browser.isDisposed()) return;
```
Comment (IlyaMikhailovHB) [inline bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/IE.java]: why we added this check on line 957 and not above?

**Comment by hb91546** [comment]
will you need to update the versions in the manifest and pom files?

File: bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/IE.java (hunk 1 above)
Comment (OSKozlov) [reply bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/IE.java]: done.

**Review by hb91546 — APPROVED** [review]

**Review by hb91546 — APPROVED** [review]

**Review by IlyaMikhailovHB — APPROVED** [review]
Tested on my sandbox. No exception

**Pull request merged**